### ISSUE TITLE
Determine analysis result variations with house events

### ISSUE THREAD

**Issue by troeger** [troeger/fuzzed#79]
A house event represents a condition that can be either TRUE or FALSE, such as planned maintenance shutdowns. For this reason, it would be interesting to have both situations (house event on / off) as separate analysis results.
Please note that this is different from configurations, which represent the addition / removal of fault tree parts.

**Comment by rakhimov** [comment]
In general, this leads to combinatorial explosion of possible "states" the system can be in,
so may the analysis be intractable.
Instead of speculatively analyzing 'on/off' states,
it is left up to event-tree layer to precisely specify under which conditions/initiating/functional events
these house events take certain values.
The planned maintenance or alignment (mission phases) can also specify its rules
to change the states of house events.

All these manipulations (and more) are specified in the Open-PSA Model Exchange Format.

In short, I think this issue should be closed in favor of Event trees.

**Comment by troeger** [comment]
Open-PSA MEF may relate event and fault trees for the purpose of performing a PSA, but this is not the only use case for the editor. The relation between modeling formats (event trees, fault trees) should not be mandatory for the purpose of having a usable analysis run.
